### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by lucabenaventew**
# Project Updates :green_circle: 
- [ ] Improve code quality

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,97 @@
...
+
+### Setup
+
+Clone the repository using the GitHub link provided below.
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
```
Comment (VICTOR-CHUKWU) [inline README.md]: - [ ] Nice work Team, kindly add a video of your presentation where you explain how you code runs and the part you are proud of  here or on the PR the video should be 5 min long maximum :+1:

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @lucabenaventew  and @asnganje , 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
## Highlights
- [ ] Linters passed
- [ ] Correct Git flow
- [ ] Clean code base

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md (hunk 1 above)
Comment (lucabenaventew) [reply README.md]: :+1:  Done

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

In my opinion, your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**